.github/workflows/selfcheck.yml

```diff
@@ -155,12 +155,12 @@ jobs:
 
       - name: Fetch corpus
         run: |
-          wget https://github.com/danmar/cppcheck/archive/refs/tags/2.8.tar.gz
-          tar xvf 2.8.tar.gz
+          wget https://github.com/danmar/cppcheck/archive/refs/tags/2.17.0.tar.gz
+          tar xvf 2.17.0.tar.gz
```

Comment:
Please revert this. This is supposed to be a fixed corpus so things can be tracked over time.